<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgInformation</class>
 <widget class="QWidget" name="wdgInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wdgInformation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLCDNumber" name="lcdRealTimeHours">
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>88.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblRealTimeSep1">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string notr="true">:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdRealTimeMinutes">
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>88.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblRealTimeSep2">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string notr="true">:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdRealTimeSeconds">
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>88.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblRealTimeText">
         <property name="text">
          <string>real time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLCDNumber" name="lcdOMSITimeHours">
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>88.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblOMSITimeSep1">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string notr="true">:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdOMSITimeMinutes">
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>88.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblOMSITimeSep2">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string notr="true">:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdOMSITimeSeconds">
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="value" stdset="0">
            <double>88.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>88</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lblOMSITimeText">
         <property name="text">
          <string>OMSI time</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lblMap">
       <property name="text">
        <string>Map:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledMap">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
